src/components/ProgressBar/index.tsx

```diff
@@ -77,12 +91,20 @@ export interface ProgressBarProps {
     resultButtonClick?: () => void;
     /** Indicate what type of progress bar to render */
     variant?: 'linear' | 'circular';
+    /** arbitrary properties to pass through */
+    props?: {
+        circularProps?: CircularProgressProps;
+        linearProps?: LinearProgressProps;
+    };
 }
 
 interface ProgressBarComponentProps {
     value: number;
     displayValue: boolean;
-    props?: any[];
+    props?: {
```

Comment:
You can also use the extends Partial<...> and then use the ...props in the call so that it is much cleaner

Reply:
I experimented with that although it has to be in a different 'namespace' otherwise you will get conflicts with properties defined in this property for example: I define variant as circular/linear wheras they base components have different definitions which results in build failures.

Reply:
You may use the [Omit Type](https://www.typescriptlang.org/docs/handbook/utility-types.html)

Reply:
Updating to use Omit. Good feedback Jessie :)